doc/classes/Engine.xml

```diff
@@ -4,16 +4,16 @@
 		Provides access to engine properties.
 	</brief_description>
 	<description>
-		The [Engine] singleton allows you to query and modify the project's run-time parameters, such as frames per second, time scale, and others.
+		The [Engine] singleton allows you to query and modify the project's run-time parameters, such as frames per second, time scale, and others. It also stores information about the current build of the Godot Engine, such as the current version.
```

Comment:
"of the Godot Engine" is something I stumbled over here, maybe "the current build of Godot" or "the current build of the engine"? Not sure. Not that important, I guess :)